include/ring-core/base.h

```diff
@@ -118,9 +124,13 @@
 // `ring::c` would need to be customized on any platform where these assertions
 // fail. Keep in sync with `ring::c`.
 OPENSSL_STATIC_ASSERT(sizeof(int32_t) == sizeof(int), "int isn't 32 bits.");
-OPENSSL_STATIC_ASSERT(sizeof(uint32_t) == sizeof(unsigned int), "unsigned int isn't 32 bits.");
-OPENSSL_STATIC_ASSERT(sizeof(size_t) == sizeof(uintptr_t), "uintptr_t and size_t differ.");
-OPENSSL_STATIC_ASSERT(sizeof(size_t) <= sizeof(uint64_t), "size_t is larger than uint64_t.");
-OPENSSL_STATIC_ASSERT(sizeof(size_t) >= sizeof(uint32_t), "size_t is smaller than uint32_t.");
+OPENSSL_STATIC_ASSERT(sizeof(uint32_t) == sizeof(unsigned int),
+                      "unsigned int isn't 32 bits.");
+OPENSSL_STATIC_ASSERT(sizeof(size_t) == sizeof(uintptr_t),
+                      "uintptr_t and size_t differ.");
+OPENSSL_STATIC_ASSERT(sizeof(size_t) <= sizeof(uint64_t),
+                      "size_t is larger than uint64_t.");
+OPENSSL_STATIC_ASSERT(sizeof(size_t) >= sizeof(uint32_t),
+                      "size_t is smaller than uint32_t.");
```

Comment:
Is this just a formatting change? If so, I'd recommend leaving it to match the upstream project, to avoid future merge conflicts.